packages/tsx/package.json

```
@@ -27,7 +27,7 @@
     "@nodejs-loaders/parse-filename": "1.0.0"
   },
   "peerDependencies": {
-    "esbuild": "~0.19.5"
+    "esbuild": "^0.25.0"
```

This is a quasi-breaking change. Instead, we should allow a range from old to new—and also confirm the APIs we need from esbuild have not changed. That's on my agenda for this week.

---

hmm but before 25 there are a security issue

---

I know ;) the security issue is irrelevant to us (we don't use the web server); if the consumer is already using esbuild though, it could affect them, and our peerDep could be preventing them upgrading.

I expect there is no issue supporting 0.25, so we should be able to list a range no problem.

A consumer _should_ upgrade esbuild, but their choice to do or not do is not our prerogative.

---

oh, actually, tsx loader's tests confirm the update doesn't break us, so we're good.

```suggestion
    "esbuild": "~0.25.0 - ~0.19.5"
```

ESBuild is pre-release, so we have to use `~`, not `^` because minors are majors in pre-release.